src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -567,9 +567,10 @@ actionFrontControllerSetVariables
     ```php
       <?php
       array(
-        'templateVars' => &(array)
+        'templateVars' => &(array) template variables
```

Comment:
```suggestion
        'templateVars' => &(array) // template variables
```

Reply:
look actionPaymentCCAdd : 'paymentCC' => (object) OrderPayment object
there is no comment anywhere else. Why do you want this ?

Reply:
I don't think this comment is necessary, unless it has to be a specific structure

Reply:
Agreed with @kpodemski 
The variable name is "templateVars" so we already have the information that it contains template variables.